### PR TITLE
Add xudeng to the OWNERS file

### PR DESCRIPTION
**Pull request by d8660091**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -29,3 +29,4 @@ approvers:
 - vignesh-goutham
 - vincentni
 - vivek-koppuru
+- d8660091
```
Comment (TerryHowe) [inline OWNERS:32]: Can you put this in there asciibetically?

**Review by TerryHowe — CHANGES_REQUESTED** [review]

**Comment by TerryHowe** [comment]
[Covere](https://github.com/aws/eks-anywhere-build-tooling/pull/1432)

File: OWNERS (hunk 1 above)
Comment (d8660091) [reply OWNERS:32]: oops, thanks for correcting it!

**Pull request closed without merging**